docs/resources/README.md

```diff
@@ -1,125 +1,598 @@
+<!-- omit in toc -->
+# Resources
+
+Whether you're completely new to the world of open source or you feel ready to tackle advanced topics like deep learning, we have plenty of resources! In addition to consulting the online resources, feel free to reach out to mentors or coordinators as well! Please note not all of these technologies are open source but they are free (for the most part).
+
+<!-- omit in toc -->
+## Table of Contents
+
+- [RCOS Essentials](#rcos-essentials)
+  - [Git / Github](#git--github)
+  - [Everything Open Source](#everything-open-source)
+    - [Open Source Leaders](#open-source-leaders)
+    - [Licensing](#licensing)
+- [Programming Languages](#programming-languages)
+  - [Python](#python)
+  - [JavaScript](#javascript)
+  - [Java](#java)
+  - [C++](#c)
+  - [C](#c-1)
+  - [PHP](#php)
+  - [Ruby](#ruby)
+  - [Swift](#swift)
+  - [TypeScript](#typescript)
+  - [C#](#c-2)
+  - [Go](#go)
+  - [Scala](#scala)
+  - [Kotlin](#kotlin)
+  - [Rust](#rust)
+- [Databases](#databases)
+  - [SQL / MySQL](#sql--mysql)
+  - [PostgreSQL](#postgresql)
+  - [MongoDB](#mongodb)
+  - [Cassandra](#cassandra)
+  - [Supabase](#supabase)
+- [Frameworks and Platforms](#frameworks-and-platforms)
+  - [JavaScript Frameworks](#javascript-frameworks)
+    - [React](#react)
+    - [Angular](#angular)
+    - [Node.js](#nodejs)
+    - [Vue.js](#vuejs)
+    - [Express.js](#expressjs)
+    - [Svelte](#svelte)
+  - [Python Frameworks](#python-frameworks)
+    - [Django](#django)
+    - [Flask](#flask)
+    - [PyTorch](#pytorch)
+    - [TensorFlow](#tensorflow)
+  - [PHP Frameworks](#php-frameworks)
+    - [Laravel](#laravel)
+    - [CodeIgniter](#codeigniter)
+    - [Symfony](#symfony)
+  - [Java Frameworks](#java-frameworks)
+    - [Spring](#spring)
+    - [Hibernate](#hibernate)
+    - [JavaFX](#javafx)
+  - [Other Frameworks](#other-frameworks)
+    - [Ruby On Rails](#ruby-on-rails)
+    - [Flutter](#flutter)
+    - [Ionic](#ionic)
+    - [Qt](#qt)
+    - [Electron](#electron)
+- [IDEs and Code Editors](#ides-and-code-editors)
+  - [Visual Studio](#visual-studio)
+  - [Eclipse](#eclipse)
+  - [IntelliJ IDEA](#intellij-idea)
+  - [Xcode](#xcode)
+  - [Android Studio](#android-studio)
+  - [Vim](#vim)
+  - [Visual Studio Code](#visual-studio-code)
+  - [Sublime Text](#sublime-text)
+  - [Notepad++](#notepad)
+- [Graphics and Design](#graphics-and-design)
+  - [GIMP](#gimp)
+  - [Blender](#blender)
+  - [Inkscape](#inkscape)
+  - [Pencil2D](#pencil2d)
+- [Hardware](#hardware)
+  - [Arduino](#arduino)
+  - [Raspberry Pi](#raspberry-pi)
+  - [BeagleBoard](#beagleboard)
+- [Diversity in STEM](#diversity-in-stem)
+  - [People of Color in Technology](#people-of-color-in-technology)
+  - [Women in Technology](#women-in-technology)
+  - [LGBTQ+ in Technology](#lgbtq-in-technology)
+- [Miscellaneous](#miscellaneous)
+  - [Amazon Web Services](#amazon-web-services)
```

Comment:
can you have a section for cloud: azure, aws, gcp, firebase, ibm cloud, oracle cloud, etc

Reply:
Added